devicetypes/qubino/child-temperature-sensor.src/child-temperature-sensor.groovy

```diff
@@ -0,0 +1,53 @@
+/**
+ *	Copyright 2020 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Child Temperature Sensor", namespace: "qubino", author: "SmartThings", mnmn: "SmartThings", vid: "generic-temperature-measurement", ocfDeviceType: "oic.d.thermostat") {
+		capability "Actuator"
```

Comment:
this isn't actually an actuator, since it doesn't _do_ anything, right?

Reply:
Good point, that capability shouldn't be there. Fixed.